pkg/translator/prometheusremotewrite/metrics_to_prw.go

```diff
@@ -27,8 +27,7 @@ type Settings struct {
 }
 
 // FromMetrics converts pmetric.Metrics to Prometheus remote write format.
-func FromMetrics(md pmetric.Metrics, settings Settings) (map[string]*prompb.TimeSeries, error) {
-	c := newPrometheusConverter()
+func (c *PrometheusConverter) FromMetrics(md pmetric.Metrics, settings Settings) (map[string]*prompb.TimeSeries, error) {
```

Comment:
Now that users create a struct before calling FromMetrics, we should consider moving any configuration or settings to the creation of the converter to simplify usage.

Reply:
This makes sense, but now it's in global sync.Pool rather than tied to a specific exporter instance so it needs to be reusable across instances.